corehq/apps/cloudcare/api.py

```diff
@@ -183,8 +184,13 @@ def keys():
                 for bool in status_to_closed_flags(self.status):
                     yield [self.domain, owner_id, bool]
 
-        view_results = CommCareCase.view('hqcase/by_owner', keys=keys,
-                                         include_docs=False, reduce=False)
+        view_results = CommCareCase.view(
+            'hqcase/by_owner',
+            keys=keys,
+            include_docs=False,
+            reduce=False,
+            stale=settings.COUCH_STALE_QUERY
+        )
```

Comment:
Yeah it would also be bad in the situation where you just created a case, and now you want to do a follow up form on it. If the thing is minutes behind, the case you just created doesn't show up and that's super confusing!